src/main/java/bio/terra/cli/context/utils/Logger.java

```diff
@@ -0,0 +1,92 @@
+package bio.terra.cli.context.utils;
+
+import bio.terra.cli.context.GlobalContext;
+import ch.qos.logback.classic.Level;
+import ch.qos.logback.classic.LoggerContext;
+import ch.qos.logback.classic.encoder.PatternLayoutEncoder;
+import ch.qos.logback.classic.spi.ILoggingEvent;
+import ch.qos.logback.core.ConsoleAppender;
+import ch.qos.logback.core.FileAppender;
+import ch.qos.logback.core.OutputStreamAppender;
+import ch.qos.logback.core.filter.Filter;
+import ch.qos.logback.core.spi.FilterReply;
+import ch.qos.logback.core.util.StatusPrinter;
+import org.slf4j.LoggerFactory;
+
+/** Utility methods for logging. */
+public class Logger {
+  private static String LOG_FORMAT = "%d{HH:mm:ss.SSS} [%thread] %-5level %logger{50} - %msg%n";
+
+  private final GlobalContext globalContext;
+
+  /**
+   * Constructor for class that sets up logging.
+   *
+   * @param globalContext the global context object
+   */
+  public Logger(GlobalContext globalContext) {
+    this.globalContext = globalContext;
+  }
+
+  /**
+   * Setup a file and console appender for the root logger. Each may use a different logging level,
+   * as specified in the global context.
+   */
+  public void setupLogging() {
+    LoggerContext loggerContext = (LoggerContext) LoggerFactory.getILoggerFactory();
+
+    // turn on logging for the logger for errors only
+    StatusPrinter.printIfErrorsOccured(loggerContext);
+
+    // build the file appender
```

Comment:
Is rolling the logs a concern for the CLI, or do we expect them to be pretty self-limiting size-wise due to the usage model?

Reply:
Good point. I think they'll be pretty self-limiting due to usage, but it's probably better to roll the files just in case. I'll take a stab at it.